TouchPortalAPI/__init__.py

```diff
@@ -286,15 +304,16 @@ def stateUpdateMany(self, states:list):
         except TypeError:
             raise TypeError(f'StateUpdateMany() requires an iteratable, got {type(states)} instead.')
 
+
     def updateActionData(self, instanceId:str, stateId:str, minValue, maxValue):
         '''
         TouchPortal currently only supports data.type "number"
         '''
-        self.send({"type": "updateActionData", "instanceId": instanceId, "data": {"minValue": minValue, "maxValue": maxValue, "id": stateId, "type": "number"}})
+        self.__send({"type": "updateActionData", "instanceId": instanceId, "data": {"minValue": minValue, "maxValue": maxValue, "id": stateId, "type": "number"}})
 
-    def send(self, data):
+    def __send(self, data):
```

Comment:
`send()` has been public API so this is a breaking change.  And 👎🏼 for changing it because the whole point is that one could send arbitrary messages to some TP version/feature which may or may not be supported by the Python API yet, and also for just using the client directly w/out the convenience methods (eg. to bypass cache or testing or whatever).

Reply:
Oh